sources/packages/backend/package-lock.json

```diff
@@ -12389,4 +12389,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
do we need this change?

Reply:
I really  didn't want to add it. But whenever I ran the code(npm install && npm run start) taking upstream from main, I saw it getting updated, Hence I added.